### ISSUE TITLE
ignore template literals for avoid-leaking-state-in-components

### ISSUE THREAD

**Issue by amk221** [ember-cli/eslint-plugin-ember#159]
```javascript
Component.extend({
  layout: hbs`<h1>foo</h1>` // Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties
});
```

**Comment by rwjblue** [comment]
Agreed, seems like a bug to me.

The fix would be to add a check for `isTemplateLiteral` to [this conditional](https://github.com/ember-cli/eslint-plugin-ember/blob/cd3fd2e58b49143ac2a2bd0c8ab2440949c511f3/lib/rules/avoid-leaking-state-in-ember-objects.js#L21-L24).

**Comment by amk221** [comment]
Ah, to clarify, I meant tagged templates. I should use `isTaggedTemplateExpression`?
PR coming
